[DOCS] Update detection rule exceptions export and import (#1434)
elastic/security-docs: docs/detections/rules-ui-manage.asciidoc

.. Drag and drop the file that contains the detection rules.
+
NOTE: Imported rules must be in an `.ndjson` file.

> .. Select *Overwrite existing detection rules with conflicting "rule_id"* to update existing rules if they match the `rule_id` value of any rules in the import file. Configuration data included with the rules, such as actions, are also overwritten.

Reviewer: ```suggestion
.. Select *Overwrite existing detection rules with conflicting "rule_id"* to update existing rules if they match the `rule_id` value of any rules in the import file. Configuration data included with the rules, such as actions, is also overwritten.
```

> .. Select *Overwrite existing exception lists with conflicting "list_id"* to replace existing exception lists with exception lists from the import file if they have a matching `list_id` value.

Reviewer 2: Are these overwriting actions optional? Seems like they might be, but it also feels as though we're encouraging users to select these options--sort of like a best practice when importing rules.

Author: Yes, these are optional check boxes. I think adding "(Optional)" to each step is an elegant solution.

> .. Select *Overwrite existing exception lists with conflicting "list_id"* to update existing exception lists associated with rules if they match the `list_id` value of any rules in the import file.

Reviewer 3: I think saying that they are "updated" if overwrite is selected is not totally accurate here. What we do behind the scenes if we find a list with a matching `list_id` is delete the existing one and it's items, then create the list using the imported one.

Author: Thanks @yctercero for the context. How about this:
```suggestion
.. Select *Overwrite existing exception lists with conflicting "list_id"* to replace existing exception lists with exception lists from the import file if they match the `list_id` value.
```

Reviewer 3: Sounds good!
